### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review 
image

### CODE REVIEW

**Review by ashishkatiyar2404 — APPROVED** [review]
All looks good to me.
it will look good if the placeholder of Email and password should not be in middle.

File: Signup.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css">
+  <link rel="stylesheet" href="https://canvas-component-lib.netlify.app/Component.css">
+  <script src="https://use.fontawesome.com/fcb99f5e0a.js"></script>
+  <title>Ecomm|Signup</title>
+</head>
+
+<body>
+  <div class="signup-wrapper">
+    <nav class="ecomm-navbar">
+      <a class="logo-img-initial" href="Signup.html">
```
Comment (gadgetfather) [inline Signup.html]: You can give better class name here.

**Comment by gadgetfather** [comment]
You can align search bar at the center
image

**Review by gadgetfather — APPROVED** [review]
Approving with suggested changes.
Great job!

**Pull request merged**